alpine/Dockerfile

```diff
@@ -16,5 +16,6 @@ RUN apk --no-cache add \
     ln -s /lib /lib64 && \
     gcloud config set core/disable_usage_reporting true && \
     gcloud config set component_manager/disable_update_check true && \
-    gcloud config set metrics/environment github_docker_image
+    gcloud config set metrics/environment github_docker_image && \
+    CLOUDSDK_COMPONENT_MANAGER_DISABLE_UPDATE_CHECK=true gcloud --version
```

Comment:
we have this set above, do we need again?

Reply:
Good call, I didn't notice that; looks like the environment variable isn't needed